content/programming-cookbook/tips-and-tricks/json-to-csharp-classes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: JSON to C# Classes
+showMetadata: true
+editable: true
+showToc: true
+---
+
+# Visual Studio
+
+- Example JSON data
+```json
+{
+    "id": 1,
+    "name": "Peter",
+    "lastname": "Parker"
+}
+```
+
+- Create your .cs class
+- Copy your JSON data
+- Select Edit => Paste Special => Paste JSON As Classes
+![Paste JSON As Classes](images/json-to-csharp-classes.png)
+
+
+- Result
```

Comment:
I also suggest using small heading instead of bullet for explain that this is result of paste JSON as C# classes.

For example:
```md
## Result
```